pkg/controller/install/install_controller.go

```diff
@@ -166,9 +169,15 @@ func (r *ReconcileInstall) install(instance *tektonv1alpha1.Install) error {
 	filters := []mf.FilterFn{
 		mf.ByOwner(instance),
 	}
-
 	r.manifest.Filter(filters...)
-	return r.manifest.ApplyAll()
+	rc := r.manifest.ApplyAll()
+
+	for _, path := range instance.Spec.AddOns {
+		extension := mf.NewYamlManifest(filepath.Join("deploy", "resources", path), true, r.config)
+		extension.Filter(filters...)
+		extension.ApplyAll()
```

Comment:
receive the return value here and return it if `err != nil`

Reply:
I'll add it.